docs/admin/install.rst

```diff
@@ -0,0 +1,336 @@
+      MD5 digest: OK
+
+
+- If the package verification was successful, in the ``dom0`` terminal, run the following command to transfer the RPM package to dom0 and install SecureDrop Workstation:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io work "cat /home/user/securedrop-workstation-dom0-config-<versionNumber>-1.fc25.noarch.rpm" > securedrop-workstation.rpm
+    sudo dnf install securedrop-workstation.rpm
+
+  When prompted, press **Y** to install the package.
+
+- Shut down the ``work`` VM using the Qube widget in the top-right panel.
+ 
+Configure SecureDrop Workstation (estimated wait time: 60-90 minutes)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before setting up the set of VMs used by SecureDrop Workstation, you must configure the Journalist Interface connection and submission key. 
+
+- To add the submission key, run the following command in the ``dom0`` terminal:
+
+  .. code-block:: sh
+
+    sudo cp /tmp/sd-journalist.sec /usr/share/securedrop-workstation-dom0-config/
+
+- Then, check its fingerprint with the command:
+
+  .. code-block:: sh
+
+    gpg --with-fingerprint --with-colons /tmp/sd-journalist.sec
+
+- Next, create the SecureDrop Workstation configuration file:
+
+  .. code-block:: sh
+   
+    cd /usr/share/securedrop-workstation-dom0-config
+    sudo cp config.json.example config.json
+
+- The ``config.json`` file must be updated with the correct values for your instance. Open it with root privileges in a text editor such as ``vi`` or ``nano`` and update the following fields' values:
+
+  - **submission_key_fpr**: use the value of the submission key fingerprint as displayed above
+  - **hidserv.hostname**: use the hostname of the Journalist Interface, including the ``.onion`` TLD
+  - **hidserv.key**: use the value of the v2 HidServAuth token for the Journalist Interface, or the v3 private authorization key value if your SecureDrop instance uses v3 onion services
+  - **environment**: use the value ``prod``
+
+.. note:: 
+  You can find the values for the **hidserv.*** fields in the ``/tmp/journalist.txt`` file that you created in ``dom0`` earlier:
+
+  If your instance uses v2 onion services, the file will be formatted as follows:
+
+  .. code-block:: none
+
+    HidServAuth ONIONADDRESS.onion AUTHTOKEN # comments, can be ignored
+
+  If your instance uses v3 onion services, the file will be formatted as follows:
+  
+  .. code-block:: none
+  
+   ONIONADDRESS:descriptor:x25519:AUTHTOKEN
+
+- Verify that the configuration is valid using the command below in the ``dom0`` terminal:
+
+  .. code-block:: sh
+
+    securedrop-admin --validate
+
+- Finally, in the ``dom0`` terminal, run the command:
+
+  .. code-block:: sh
+
+    securedrop-admin --apply
+
+This command will take a considerable amount of time and approximately 4GB of bandwidth, as it sets up multiple VMs and installs supporting packages. When it completes, your SecureDrop Workstation is finally ready to use!
```

Comment:
The most common reason I'm aware of that this command may fail on first run are network issues in guest VMs even when the host network is up. Both Mickael and I have observed this in recent runs. In case this happens, a re-run is required. I'll open an issue to track, and if we don't resolve soon, we may want to add a note here.